app/League-Entries/page.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { LeagueEntries } from '@/components/LeagueEntries/LeagueEntries';
+import { LeagueSurvivors } from '@/components/LeagueSurvivors/LeagueSurvivors';
+import * as React from 'react';
```

Comment:
Do not import all of React's dependencies. Only import what is needed. In this case, it's `React, { JSX } from 'react'`

Reply:
Updated in latest commit.